<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkullCrack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="10" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Willies 0-7:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Pharts 0-7:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="levelComboBox"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>1970's 0-3:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Lives 0-99:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Level:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QPlainTextEdit" name="outputEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Birds 0-7:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Enemas 0-7:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLineEdit" name="_1970sEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="phartsEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QPushButton" name="generateButton">
      <property name="text">
       <string>Generate Password</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="enemasEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="birdsEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLineEdit" name="williesEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="livesEdit">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
